### DISCUSSION TITLE
Remove / add fields to filter in contacts

### DISCUSSION THREAD

**Discussion by OscarGS86** [YetiForceCompany/YetiForceCRM#15358] [How to] [answered]
Hi guys,

Do you know how I can remove or add fields here to allow me to filter by a city or any extra field?

![Screenshot 2021-10-25 at 10 12 49](https://user-images.githubusercontent.com/92228499/138659039-c34b9bf1-c75e-4fd6-b02e-de42a662d6a8.png)

Thank you!

**Comment by rskrzypczak** [comment, marked as answer]
List view:
/Software configuration/Labels, searching and filtering/Filters - Configuration
(https://gitdeveloper.yetiforce.com/index.php?module=CustomView&parent=Settings&view=Index)

for a list of related records:
/Software configuration/Standard modules/Modules – Relations
(https://gitdeveloper.yetiforce.com/index.php?module=LayoutEditor&parent=Settings&view=Index&mode=showRelatedListLayout)

**Comment by OscarGS86** [comment]
Hi @rskrzypczak ,

Is this not controlled by the field configuration? I just need to add the city field in the header to allow me to apply a filter as I can do now for First Name, Last Name, Primary email, Office phone or Assigned to. Could I add a new field on the header?

Thanks for your help

**Reply by rskrzypczak** [reply]
> Is this not controlled by the field configuration?

No. You have to edit your Custom view(filter), e.g. from the places I mentioned previously.
